swagger.yaml

```diff
@@ -418,7 +418,18 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/UpdateUserAttribute"
+              type: object
+              description: Request to update a user's attribute
+              required:
+                - id
+                - attributes
+              properties:
+                id:
+                  type: string
+                  format: uuid
+                  description: User's ID in Keycloak.
+                attributes:
+                  $ref: "#/components/schemas/Attributes" 
       responses:
         "200":
           description: User attributes updated successfully  
```

Comment:
Github won't let me leave a comment on the specific line unfortunatly but on line 439 (the schema ref) rename updateUserAttributeResponse to UpdateUserAttribute